frontends/parsers/p4/p4parser.ypp

```diff
@@ -157,6 +157,9 @@ inline std::ostream& operator<<(std::ostream& out, const P4::Token& t) {
                    START_INTEGER_TRIPLE
                    START_STRING_LITERAL_TRIPLE
 
+                   // P4Runtime annotations
+                   START_P4RT_TRANSLATION_ANNOTATION  // @p4runtime_translation
```

Comment:
@mbudiu-vmw I think the feedback we are looking for here is whether you are ok with "hardcoding" support for this annotation in the P4 parser? Or would you prefer a different approach? We could treat is as an unstructured annotation and "parse" it dynamically using C++ regular expressions. Would like to have and opinion from @liujed @liujed-intel as well if he can think of a better approach.

Reply:
Sure, this is fine. If this is a "standard" annotation this is a reasonable way to do it.

Reply:
Agreed, this is fine if we expect all P4 compilers to support this annotation.